guide/migration-from-riot2.md

```diff
@@ -41,12 +41,16 @@ Notice that now `data-is` could be used also in nested tags and it supports dyna
 ### Deprecate `name` and `id` to cache DOM nodes
 
 The attributes `name` and `id` will be no longer used by riot to cache DOM references to the tag context.
-Use the `ref` attribute instead
+Use the `ref` attribute instead.
+children tags are now targeted via `ref` too.
+This does not rename them in the `this.tags` object anymore, but adds them as tag instance to `this.refs`.
```

Comment:
Thanks. I got the point... Wrap up:

"Use the `ref` attribute instead. You will get the references to DOMs or custom tags through `this.refs`. On the other hand, note that `this.tags` will have no named tags anymore now."

Anyway, we need a native checker, I think. @rsbondi could you check these?